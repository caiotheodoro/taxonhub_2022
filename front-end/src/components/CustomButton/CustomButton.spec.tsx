import { render } from "@testing-library/react";
import { FiHome } from "react-icons/fi";
import { CustomButton } from "./index";


describe("NavItem components", () => {
  it("should render correctly", () => {
    const { container } = render(<CustomButton >teste</CustomButton>);

    expect(container).toMatchSnapshot();
  });

  it("should render a icon", () => {
    const { container } = render(<CustomButton icon={FiHome}>teste</CustomButton>);

    expect(container.querySelector("svg")).toBeInTheDocument();
  });
});
