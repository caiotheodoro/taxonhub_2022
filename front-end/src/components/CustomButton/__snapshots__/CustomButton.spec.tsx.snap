// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem components should render correctly 1`] = `
<div>
  <button
    class="chakra-button css-e3tkjo"
    type="button"
  >
    teste
  </button>
</div>
`;
